features/orabos/exec.post

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
this copy could be neglected if using file-include

Reply:
See PR https://github.com/gardenlinux/gardenlinux-ccloud/pull/13